tutorials/platform/android/android_in_app_purchases.rst

```diff
@@ -13,177 +13,157 @@ Getting started
 ~~~~~~~~~~~~~~~
 
 Make sure you have enabled and successfully set up :ref:`Android Gradle Builds <doc_android_gradle_build>`.
-Follow the compiling instructions on the ``GodotGooglePlayBilling`` `github page <https://github.com/godotengine/godot-google-play-billing>`__.
-
-Then put the files `./godot-google-play-billing/build/outputs/aar/GodotGooglePlayBilling.***.release.aar` and `./GodotGooglePlayBilling.gdap` into your project in the `res://android/plugins` folder.
-
-The plugin should now show up in the Android export settings, where you can enable it.
+Follow the installation instructions on the ``GodotGooglePlayBilling`` `github page <https://github.com/godotengine/godot-google-play-billing>`__.
 
 
 Initialize the plugin
 ~~~~~~~~~~~~~~~~~~~~~
 
 To use the ``GodotGooglePlayBilling`` API:
 
-1. Obtain a reference to the ``GodotGooglePlayBilling`` singleton
-2. Connect handlers for the plugin signals
-3. Call ``startConnection``
+1. Access the ``BillingClient`` autoload singleton, it's automatically added when the plugin is enabled.
+2. Connect to its signals to receive billing results
+3. Call ``start_connection``
```

Comment:
```suggestion
1. Access the ``BillingClient`` autoload singleton, it's automatically added when the plugin is enabled.
2. Connect to its signals to receive billing results.
3. Call ``start_connection``.
```
Smallest nitpick, but either all of them should end with `.`, or none of them IMO. Otherwise, this looks good to me.